zerver/tests/test_subs.py

```diff
@@ -402,6 +402,36 @@ def test_admin_remove_others_from_unsubbed_private_stream(self):
         self.assert_json_error(
             result, "Cannot administer invite-only streams this way")
 
+    def test_create_invite_only_stream_owner(self):
+        # type: () -> None
+        """
+        When a user creates a invite only stream, create him/her
```

Comment:
Awkward phrasing. I'd make it something like "A user who creates an invite only stream should by default be the owner of that stream."